### ISSUE TITLE
Performance results?

### ISSUE THREAD

**Issue by tejaskhot** [JamesChuanggg/vqa-tf#1]
Can you post results of how it compares to the original implementation and if you're achieving the same level of accuracy.

**Comment by JamesChuanggg** [comment]
Hi, the current code can reach 49.49 on Open-Ended (use COCO train set to train and validation set to evaluation). I'm still trying to improve its performance. Please let me know if you raise its accuracy in any way.

**Comment by chingyaoc** [comment]
Hi, I fix my code and it can now reach same accuracy with original torch code on Open-Ended val set.
